apertium-eng.eng.rlx

```diff
@@ -329,6 +344,33 @@ SELECT V IF (-1 (prn pers)) ;
 SELECT SUB:1 Be IF (1 Ing) (0/* Have) ;
 SELECT Ger IF (-1/* Be) (1 To) ;
 
+SELECT Adv IF (0 Pr) (-1 Vblex) ;
+#Example (on):
+#He talked on.
```

Comment:
Has this been addressed?

Reply:
I can delete this rule, but I tested some sentences, and they seemed to work.  

`He fell out the window` doesn't seem to work, because `out` only seems to be listed as an adverb.

The other two examples did work, though.

I also tested two other words: `across and by`, both of which are listed as adverbs and prepositions.

`She jumped across/He ran across the lawn`
`He waited on/She waited on the train`

These worked correctly. 

I can still delete the rule, though, if you'd like.

Reply:
Maybe I can put SELECT Adv IF (0 Pr) (-1 Vblex) (1 Sent) ;
or something.  That would work, though it would require relying on punctuation.

Reply:
The only reason things like "He ran across the lawn" works is by chance.  Definitely remove the rule.

Reply:
Sure, I'll do that.